src/utils/filter_keyword_utils.js

```diff
@@ -39,7 +37,7 @@ export async function removeSearchTerm(
   return;
 }
 
-//   Creates a "Tag" in "filters" section for each keyword searched
+//*   Creates a "Tag" in "filters" section for each keyword searched
 export function processFilters(filters) {
   let res = [];
   for (const i in filters) {
```

Comment:
The github ui won't allow me to comment at the bottom because no changes were added to the bottom of the utils, but I would add an empty line at the end of each file as this is an industry standard!

Reply:
Noted, thanks!